examples/jsm/loaders/PLYLoader.js

```diff
@@ -343,46 +345,31 @@ class PLYLoader extends Loader {
 
 			let result;
 
-			const patternBody = /end_header\s([\s\S]*)$/;
+			const patternBody = /end_header\s+([\s\S]*)$/;
 			let body = '';
 			if ( ( result = patternBody.exec( data ) ) !== null ) {
 
 				body = result[ 1 ];
 
 			}
 
-			const lines = body.split( /\r\n|\r|\n/ );
-			let currentElement = 0;
-			let currentElementCount = 0;
-			let elementDesc = header.elements[ currentElement ];
-			let attributeMap = mapElementAttributes( elementDesc.properties );
-
-			for ( let i = 0; i < lines.length; i ++ ) {
-
-				let line = lines[ i ];
-				line = line.trim();
-				if ( line === '' ) {
```

Comment:
Since this code block has been removed: Is the loader still able to skip over empty lines?